docs/source/fetching/defer.mdx

```diff
@@ -2,13 +2,13 @@
 title: Using the @defer directive in Apollo iOS
 ---
 
-> ⚠️ **The `@defer` directive is currently [experimental](https://www.apollographql.com/docs/resources/release-stages/#experimental-features) in Apollo iOS and is available for use in the `TBD` release.** If you have feedback on the feature, please let us know via [GitHub issues](https://github.com/apollographql/apollo-ios/issues/new?assignees=&labels=bug%2Cneeds+investigation&projects=&template=bug_report.yaml).
+> ⚠️ **The `@defer` directive is currently an [experimental feature](https://www.apollographql.com/docs/resources/release-stages/#experimental-features) in Apollo iOS and is available for use from version x.y.z.** If you have feedback on the feature, please let us know via a new [GitHub issue](https://github.com/apollographql/apollo-ios/issues/new?assignees=&labels=bug%2Cneeds+investigation&projects=&template=bug_report.yaml) or search through any [existing `@defer`-related issues](https://github.com/apollographql/apollo-ios/issues?q=is%3Aissue+is%3Aopen+%22defer%22).
```

Comment:
Let's just make sure not to merge this PR in until we are ready to release a version with defer and change that from "version x.y.z" to the appropriate version number.

Reply:
Currently this is to be merged into the `feature/defer-execution-networking` branch, which will then get merged into `main`. Two options then:
1. Redo this change against main and leave it till we know which version it will be released in.
2. Keep with the current merge path and update the doc before release.

Reply:
I think let's stick with option 2, which is the current merge path. At the point that we merge `feature/defer-execution-networking` into `main` it's ready for release so updating the docs can happen in the release branch.